### ISSUE TITLE
Typography title is overlapping with the sidebar on the small screen.

### ISSUE THREAD

**Issue by gravityNotWorking** [Mirrorful/mirrorful#181]
I know that the app is not currently mobile responsive, but even on my 1440x900 screen, the title and second title are overlapping with the sidebar, as shown.

![image](https://user-images.githubusercontent.com/77449139/226109157-e94cf8c3-3f03-4b73-b943-d714a78d7f22.png)

**Comment by alexdanilowicz** [comment]
Yep, I noticed this too. Going to fix this in an upcoming PR of mine unless someone beats me to it. Thanks for opening.
